cli.js

```diff
@@ -4,7 +4,10 @@ var electron = require('./')
 
 var proc = require('child_process')
 
-var child = proc.spawn(electron, process.argv.slice(2), {stdio: 'inherit'})
+var env = process.env
+env.ELECTRON_ENV = env.ELECTRON_ENV || 'development'
+
+var child = proc.spawn(electron, process.argv.slice(2), {stdio: 'inherit', env: env})
```

Comment:
Since the `process.env` of the current process is modified on line 8, I wouldn't think you would need to pass in the environment to `spawn` since that should inherit the current process's env by default right?